### PR TITLE
docs: Edited to remove jargon in Reference, Glossary, and Style sections

### PR DESCRIPTION
**Pull request by TeriGlover**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by petebacondarwin** [comment]
Hi @TeriGlover - this needs a rebase toi trigger the CI again.

File: aio/content/guide/glossary.md [hunk 1]
```diff
@@ -338,7 +338,7 @@ There are three categories of directive:
 
 Angular supplies a number of built-in directives that begin with the `ng` prefix.
 You can also create new directives to implement your own functionality.
-You associate a *selector* (an HTML tag such as `<my-directive>`) with a custom directive, thereby extending the [template syntax](guide/template-syntax) that you can use in your apps.
+You associate a *selector* (an HTML tag such as `<my-directive>`) with a custom directive, thereby extending the [template syntax](guide/template-syntax) that you can use in your applications.
```
Comment (petebacondarwin) [inline aio/content/guide/glossary.md:341]: `thereby` is a pretty uncommon word too??

File: aio/content/guide/styleguide.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ On the other hand, you need a really unusual case for breaking a *Do* guideline.
 <div class="s-rule consider">
 
 **Consider** guidelines should generally be followed.
-If you fully understand the meaning behind the guideline and have a good reason to deviate, then do so. Please strive to be consistent.
+If you fully understand the meaning behind the guideline and have a good reason to deviate, then do so. Strive to be consistent.
```
Comment (petebacondarwin) [inline aio/content/guide/styleguide.md:27]: `Strive` is arguably an unnecessary use of an uncommon word here. Perhaps `Aim` instead?

File: aio/content/guide/styleguide.md [hunk 3]
```diff
@@ -1636,9 +1636,9 @@ A consistent class and file name convention make these modules easy to spot and
 
 ## Application structure and NgModules
 
-Have a near-term view of implementation and a long-term vision. Start small but keep in mind where the app is heading down the road.
+Have a near-term view of implementation and a long-term vision. Start small but keep in mind where the application is heading down the road.
```
Comment (petebacondarwin) [inline aio/content/guide/styleguide.md:1639]: `down the road` is rather idiomatic. If we are striving[sic] to make the docs more accessible to non-native English speakers, should we find a more straightforward phrase here?

File: aio/content/guide/styleguide.md [hunk 4]
```diff
@@ -3195,7 +3195,7 @@ helps instantly identify which members of the component serve which purpose.
 
 
 
-**Why?** Logic may be reused by multiple components when placed within a service and exposed via a function.
+**Why?** Logic may be reused by multiple components when placed within a service and exposed by way of a function.
```
Comment (petebacondarwin) [inline aio/content/guide/styleguide.md:3198]:
```suggestion
**Why?** Logic may be reused by multiple components when placed within a service and exposed as a function.
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Thanks @TeriGlover - A couple of small suggestions.

File: aio/content/guide/glossary.md (hunk 1 above)
Comment (TeriGlover) [reply aio/content/guide/glossary.md:341]:
```suggestion
You associate a *selector* (an HTML tag such as `<my-directive>`) with a custom directive; this extends the [template syntax](guide/template-syntax) that you can use in your applications.
```

**Review by aikithoughts — APPROVED** [review]
LGTM.

Reviewed-for: global-docs-approvers

**Review by jessicajaniuk — APPROVED** [review]
LGTM :cookie:

reviewed-for: public-api

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: public-api

**Review by alxhub — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
Note to Caretaker: this is a straightforward change in the docs that trigger `public-api` group approvals. We've got 2 out of 3, but it should be safe to proceed with the current set of approvals.

**Pull request closed without merging**